<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderStatusWindow</class>
 <widget class="QMainWindow" name="OrderStatusWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменить статус заказа</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="order_comboBox"/>
    </item>
    <item>
     <widget class="QCheckBox" name="is_completed_checkBox">
      <property name="text">
       <string>Выполнен</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="is_paid_checkBox">
      <property name="text">
       <string>Оплачен</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="save_pushButton">
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>